Weather App/UI/SearchVC/SearchView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  SearchView.swift
+//  Weather App
+//
+//  Created by Ivan Stajcer on 31.08.2021..
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class SearchView : UIView {
+    var tableView = UITableView()
+    var tableDelegate : UITableViewDelegate?
+    var tableDataDelegate : UITableViewDataSource?
```

Comment:
What is a data delegate?

Reply:
It should be a tableDataSource, instead of delegate.